examples/binderhub/traefik/rbac-config.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Maybe point to these files directly : https://github.com/cetic/fadi/tree/master/helm/traefik ? We don't need to duplicate them.

Reply:
Delete this part from the binderhub example